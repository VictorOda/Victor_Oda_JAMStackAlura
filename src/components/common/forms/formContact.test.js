import React from 'react';
import { screen } from '@testing-library/react';
import FormContact from './formContact';
import { render } from '../../../infra/tests/testUtils';

jest.mock('font-awesome/css/font-awesome.min.css', () => ({
  FontAwesomeIcon: '',
}));

describe('<FormContact />', () => {
  test('renderiza componente', () => {
    const onClose = jest.fn();
    render(
      <FormContact
        onClose={onClose}
        isOpen
      />,
    );
    screen.debug();
    const textInput = screen.getByPlaceholderText(/Seu nome aqui/i);
    expect(textInput).toMatchSnapshot();
  });
});
