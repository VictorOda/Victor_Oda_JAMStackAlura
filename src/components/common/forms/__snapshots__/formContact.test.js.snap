// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormContact /> renderiza componente 1`] = `
.c0 {
  width: 90%;
  margin: auto;
  height: 32px;
  padding: 8px;
}

@media screen and (min-width:0px) {

}

@media screen and (min-width:768px) {

}

<input
  class="c0"
  id="name"
  placeholder="Seu nome aqui"
  type="text"
  value=""
/>
`;
